cl/people_db/import_judges/populate_fjc_judges.py

```diff
@@ -445,7 +450,9 @@ def make_federal_judge(item, testing=False):
         else:
             school_str = ''
 
-        schoolname = item['Name of School' + school_str]
+        school_str = ' (%s)' % schoolnum
```

Comment:
This is weird. Revise?

Reply:
I'll pull out 448-452 in the next pull request - every field related to education has a number wrapped in a parenthetical (including "1") - but I'm not quite sure I see the issue here.  In the FJC data the variable is "Name of School (1)", "Name of School (2)" and so on.

Reply:
Just that lines 448-452 don't have any effect after the change you added line 453.

Reply:
Should I submit an updated pull request omitting 448-452 now?  They should not be included and I already have another pull request waiting in the wings after this is approved that removes these.  But I can do it now if you prefer.

Reply:
Nah, if it's already ready to go, I'll merge this and you can send that. Merging.